### DISCUSSION TITLE
716

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#898] [Comments]
# 716

요즘 스타트업 제품팀과 종종 미팅을 한다. 제품은 이미 출시되었는데, 이제 시작 단계라 계속해서 속도전으로 기능을 붙이는 팀들이다. 대부분 전체 팀원이 10인 이하이거나 개발팀원이 3 ~ 7이다. 그러다보니 고정비에 대한 고민을 항상 듣게 된다. 트래픽이나 가입자수는 아직 보잘 것 없고, 수익은 0원인데, 5명 ~ 10명의 급여로 매달 수천만원씩 나가고 있는 상황. 이런 상황에서 한달에 클라우드 사용료로 30만원이라도 나오게 되면 이걸 어떻게 아낄 수 없을까 하는 고민이 먼저 든다. (EC2, ALB, Route53, CloudFront, RDS, Redis, S3 등 최소한의 구색이라도 갖춘 경우) 물론 몇천만원씩 급여를 주니 이정도는 감안할 수 있는거 아니야? 라고 할 수 있다. 하지만… 월급 200..

https://jojoldu.tistory.com/716

**Comment by heli-os** [comment]
첫 번째 AWS Activate는 가입 시 해당 프로그램 연계로 지원해주는 SaaS가 정말 많습니다! 교육도 많이 있어서 더 좋은 것 같아요.

첫 번째 페이지만 훑어봐도 Notion $1,000, MongoDB $5,000, miro $1,000, datadog $100,000, ... 등등 다양하네요.

저희는 인프런 통해서 가입하려고 했으나 투자사가 AWS Activate $10,000 협약 투자사라 그걸 이용했고 그 후속으로 데이터독 크레딧을 정말 알차게 쓰고 있습니다.

물론 기한이 무제한은 아니라 나중에 비용으로 돌아올 것이라 소중히 쓰고 있습니다. JetBrains 50% 할인은 몰랐는데 좋은 정보 감사합니다..!

**Reply by jojoldu** [reply]
오 태양님 답변 감사합니다.
투자사가 이미 협약 투자사군요?? 대박.. ㅋㅋ 
말씀하신대로 AWS Activate 가 되면 추가로 받을 SaaS 혜택을 포함시키지 않았는데,
해당 내용도 포함시키겠습니다! ㅋㅋ 감사합니다 :)

**Comment by stella6767** [comment]
유용한 정보입니다 ㅎ. 항상 잘 보고 있어요
